hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/HddsConfigKeys.java

```diff
@@ -74,6 +74,10 @@ public final class HddsConfigKeys {
   public static final String HDDS_CONTAINER_CLOSE_THRESHOLD =
       "hdds.container.close.threshold";
   public static final float HDDS_CONTAINER_CLOSE_THRESHOLD_DEFAULT = 0.9f;
+
+  public static final String HDDS_SCM_TXN_DN_COMMIT_MAP_SIZE = "hdds.scm.txn.dn.commit.map.size";
+
+  public static final int HDDS_SCM_TXN_DN_COMMIT_MAP_SIZE_DEFAULT = 5000000;
```

Comment:
Please add new property in `ScmConfig` instead, like:

https://github.com/apache/ozone/blob/319d5a4cfed43cae2885946452145b4fc3e461e2/hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/scm/ScmConfig.java#L105-L117

Probably should include `block.deletion` in the name.  Also, it should indicate that this is a limit.

Reply:
Done.

Reply:
Thanks for the improvement.